Exiled.Example/Example.cs

```diff
@@ -41,21 +37,18 @@ private Example()
         /// <inheritdoc/>
         public override void OnEnabled()
         {
-            base.OnEnabled();
-
             RegisterEvents();
-
```

Comment:
Revert that like breaker. It looks better, you don't get confused in logs and method calls with a large number of logs.